pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/KeyValueSchemaInfoData.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.client.impl.schema;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Accessors(chain = true)
+@Builder
+public class KeyValueSchemaInfoData {
```

Comment:
@congbobo any reason creating a new class? Can't you just `KeyValue<Object, Object>`?

Reply:
> @congbobo any reason creating a new class? Can't you just `KeyValue<Object, Object>`?
yes, there should use keyValue<Object, Object>, and then i will change it and add the test for it .

Reply:
> @congbobo any reason creating a new class? Can't you just `KeyValue<Object, Object>`?

yes, there should use keyValue<Object, Object>, and then i will change it and add the test for it .